runtime/doc/lua.txt

```diff
@@ -1297,6 +1299,160 @@ validate({opt})                                               *vim.validate()*
                                 fails
 
 
+==============================================================================
+Lua module: keymap                                                *lua-keymap*
+
+cmap({opts})                                               *vim.keymap.cmap()*
+                Helper function for ':cmap'.
+
+                See also: ~
+                    |vim.keymap.nmap|
+
+cnoremap({opts})                                       *vim.keymap.cnoremap()*
+                Helper function for ':cnoremap'
+
+                See also: ~
+                    |vim.keymap.nmap|
+
+imap({opts})                                               *vim.keymap.imap()*
+                Helper function for ':imap'.
+
+                See also: ~
+                    |vim.keymap.nmap|
+
+inoremap({opts})                                       *vim.keymap.inoremap()*
+                Helper function for ':inoremap'
+
+                See also: ~
+                    |vim.keymap.nmap|
+
+lmap({opts})                                               *vim.keymap.lmap()*
+                Helper function for ':lmap'.
+
+                See also: ~
+                    |vim.keymap.nmap|
+
+lnoremap({opts})                                       *vim.keymap.lnoremap()*
+                Helper function for ':lnoremap'
+
+                See also: ~
+                    |vim.keymap.nmap|
+
+map({opts})                                                 *vim.keymap.map()*
+                Helper function for ':map'.
+
+                See also: ~
+                    |vim.keymap.nmap|
+
+nmap({opts})                                               *vim.keymap.nmap()*
+                Helper function for ':nmap'.
+>
+
+    vim.keymap.nmap { 'lhs', function() print("real lua function") end, silent = true }
+<
+
+                Parameters: ~
+                    {opts}  (table): A table with keys:
+                            • [1] = left hand side: Must be a string
+                            • [2] = right hand side: Can be a string OR a lua
+                              function to execute
+                            • Other keys can be arguments to |:map|, such as
+                              "silent". See |nvim_set_keymap()|
+
+nnoremap({opts})                                       *vim.keymap.nnoremap()*
+                Helper function for ':nnoremap' >
```

Comment:
- we never need to call anything a "helper". all functions are helpful. just say what it does. 
- single-quotes in vim docs mean "this is an option". 

```suggestion
                Interface to |:nnoremap| >
```